### PR TITLE
[IMP]: point_of_sale: terms and conditions added in invoice

### PR DESCRIPTION
**Pull request by trj-odoo**

### CODE REVIEW

File: addons/point_of_sale/models/pos_order.py [hunk 1]
```diff
@@ -467,14 +467,25 @@ def action_pos_order_paid(self):
     def _prepare_invoice_vals(self):
         self.ensure_one()
         timezone = pytz.timezone(self._context.get('tz') or self.env.user.tz or 'UTC')
+        note = self.note or ''
```
Comment (pimodoo) [inline addons/point_of_sale/models/pos_order.py:470]: @trj-odoo most of the function really looks like a copy paste from the code in sale, can't we create a funnction in account module in 'account_move.py' file called "_get_default_terms_and_condition" and call it here and in the function "_default_note" of the sale order?

Comment (trj-odoo) [reply addons/point_of_sale/models/pos_order.py:470]: yeah could do

Comment (trj-odoo) [reply addons/point_of_sale/models/pos_order.py:470]: Nvm, this PR is not relevant anymore. https://github.com/odoo/odoo/pull/73121 changed how T&C are added in the invoice. I need to adapt the code

**Comment by pimodoo** [comment]
put this as a fix in https://github.com/odoo/odoo/pull/75016 for V 14.0

**Pull request closed without merging**